src/components/Banner/Banner.tsx

```diff
@@ -52,15 +53,45 @@ const ContentWrapper = styled(Padbox)`
 
 /* stylelint-disable */
 const Text = styled(BaseText)<{ $variant?: BannerProps['variant'] }>`
-  max-width: 125ch;
+  max-width: 150ch;
   color: ${getColor('neutral.1000')};
   display: -webkit-box;
   overflow-y: auto;
```

Comment:
I think you can also remove those two lines and then remove the stylelint comment